clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/AssignmentChangeApplicationEvent.java

```diff
@@ -26,8 +26,17 @@ public class AssignmentChangeApplicationEvent extends ApplicationEvent {
     final long currentTimeMs;
 
     public AssignmentChangeApplicationEvent(final Map<TopicPartition, OffsetAndMetadata> offsets, final long currentTimeMs) {
-        super(Type.ASSIGNMENT_CHANGE);
+        super(Type.ASSIGNMENT_CHANGE, AssignmentChangeApplicationEvent.class.getSimpleName());
```

Comment:
This would then change back to how it was if we agree on using the getClass().getSimpleName() in the `toStringBase`